lib/cylc/cfgspec/globalcfg.py

```diff
@@ -127,6 +127,17 @@
             'retrieve job logs max size'  : vdr( vtype='string' ),
             'retrieve job logs retry delays': vdr( vtype='interval_minutes_list', default=[] ),
             'task event handler retry delays': vdr( vtype='interval_minutes_list', default=[] ),
+            'local tail command template' : vdr( vtype='string', default="tail -n +1 -F %(filename)s"),
+            'remote tail command template' : vdr( vtype='string', default="tail --pid=`ps h -o ppid $$ | sed -e s/[[:space:]]//g` -n +1 -F %(filename)s"),
+            # Template for tail commands on remote files.
```

Comment:
I know. It does not work on RHEL6 but OK on SuSE11. (I suspect that it is because SuSE11 has a very old version of `bash` which may be more forgiving for this sort of stuff.)